src/Protocol/SmtpPluginManager.php

```diff
@@ -9,22 +9,20 @@
 
 namespace Zend\Mail\Protocol;
 
-use Zend\ServiceManager\AbstractPluginManager;
```

Comment:
@weierophinney the get() definition in SmtpPluginManager has the optional parameter $option, that is not defined by container-interop. I know that in PHP the interface compatibility is ok with optional parameters but I want to be sure that we all agree on this "hack".